text_collectors/boots.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+#
+# Description: Expose metrics for number of boots of the system
+# Dependencies: none
+#
+# The script creates one metric series which is the number of recorded
+# boots in the /var/log/wtmp file. If wtmp is rotated, the next time the
+# node exporter is restarted it will recalculate the value, and may over
+# count restarts by one.
+#
+# Author: Clayton Coleman <smarterclayton@gmail.com>
+
+set -euo pipefail
+
+v=$(mktemp --suffix=.boots.working)
+
+reboots=$( last reboot --time-format=iso -R | grep -cE '^reboot' )
+if [[ "${reboots}" -gt 0 ]]; then
+  echo '# HELP node_boots reports a single series which is the number of times this system has been booted excluding the current boot. If the value is zero, this is the first time the system has booted. The value is always non-negative.' >>"${v}"
+  echo '# TYPE node_boots counter' >>"${v}"
+  echo "node_boots{} $(( reboots - 1 ))" >>"${v}"
```

Comment:
that should be `node_boots_total` since it's a counter (at some point Prometheus will mandate the `_total` suffix because of OpenMetrics).